syntaxes/bl4.tmLanguage.json

```diff
@@ -27,6 +44,10 @@
         }
       ]
     },
+    "type": {
+      "name": "entity.name.type",
+      "match": "(?<=\\:|->)\\s+?\\w+|\\((\\w+\\,\\s*\\w+)\\)"
```

Comment:
Can you please add a comment for this one :-) ?

Reply:
done!